src/StringUtils.h

```diff
@@ -8,7 +8,14 @@
 
 #pragma once
 
+#include <cstddef>
 
+#include <string>
+#include <list>
+#include <vector>
+#include <map>
+
+#include "fmt.h"
```

Comment:
This definitely shouldn't be necessary, all of these headers are in `Globals.h`.